x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details.js

```diff
@@ -123,14 +123,14 @@ function getDetailsItems(anomaly, examples, filter) {
   }
   items.push({
     title: i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.timeTitle', {
-      defaultMessage: 'time',
+      defaultMessage: 'Time',
```

Comment:
The original idea was that the expanded row is more of a 'raw' view of the anomaly record, which is why we didn't capitalize the field names in this section, and also went with camelCase `fieldName` for example rather than `Field name`. I prefer the camelCase field names, but maybe that is just because it is what I am used to seeing! You could also argue that the current formatting is inconsistent, as it uses `job ID` (rather than `jobId`) and `multi-bucket impact`.

![image](https://user-images.githubusercontent.com/7405507/115389563-240efc80-a1d5-11eb-97e2-e377757ab586.png)

Reply:
I can revert that change if necessary, but I think it's more consistent to use the same capitalization and formatting throughout the table.